Revert these commits:

https://github.com/ImageMagick/ImageMagick6/commit/f665638007d1f8627677a2c60ceda971c1d6cbb4
https://github.com/ImageMagick/ImageMagick6/commit/ae55feba3f47534a999fdc0c37e0ec0fb34231f1

These commits cause files not to be installed in vendor_perl anymore which
causes the perl5 portgroup to fail.

These commits also cause files to be installed in unversioned directories so
that the subports would conflict with one another.
--- PerlMagick/Makefile.PL.in.orig	2024-05-25 07:53:44.000000000 -0500
+++ PerlMagick/Makefile.PL.in	2024-06-18 21:48:55.000000000 -0500
@@ -223,9 +223,6 @@
    # Linker flags for building a dynamically loadable module
    'LDDLFLAGS' => $LDDLFLAGS_magick,
 
-   # Install PerlMagick into ImageMagick prefix
-   'INSTALL_BASE' => '@PREFIX_DIR@',
-
    # Install PerlMagick binary into ImageMagick bin directory
    'INSTALLBIN'	=> '@BIN_DIR@',
 
--- PerlMagick/quantum/Makefile.PL.in.orig	2024-06-17 19:40:10.000000000 -0500
+++ PerlMagick/quantum/Makefile.PL.in	2024-06-18 21:37:23.000000000 -0500
@@ -220,9 +220,6 @@
    # Linker flags for building a dynamically loadable module
    'LDDLFLAGS' => $LDDLFLAGS_magick,
 
-   # Install PerlMagick into ImageMagick prefix
-   'INSTALL_BASE' => '@PREFIX_DIR@',
-
    # Install PerlMagick binary into ImageMagick bin directory
    'INSTALLBIN'	=> '@BIN_DIR@',
 
